jbmc/src/java_bytecode/remove_exceptions.cpp

```diff
@@ -87,14 +87,14 @@ class remove_exceptionst
 
   explicit remove_exceptionst(
     symbol_table_baset &_symbol_table,
-    const class_hierarchyt *class_hierarchy,
```

Comment:
These changes are unnecessary, the init list knows the difference between locals and members